### PR TITLE
13/04 Dev

### PR DESCRIPTION
**Pull request by LuongHuy**

### CODE REVIEW

File: online-grocery-store/src/components/layout/MainLayout.js [hunk 1]
```diff
@@ -8,7 +9,19 @@ const Header = () => {
       <Grid.Col span={2} />
       <Grid.Col span={8}>
         <Flex h="100%" justify="space-between" align="center">
-          <Link href="/">Header</Link>
+          <Link href="/">
+            <Image radius="md" src="favicon.ico" h={40} w="auto" />
```
Comment (dustinle48) [inline online-grocery-store/src/components/layout/MainLayout.js]: Should also have the Website name after the logo as well. And please use another logo.

**Review by dustinle48 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -28,3 +28,16 @@ Open http://localhost:8080 -> use **root/root** to login -> import sql file in d
...
+
+5. Git Commit -> Git Push
+   git commit -m""
+   git push origin "branch_name"
+
+6. Change branch
+   git checkout "branch_name"
+
+7. Pull from main
+   git pull origin main
```
Comment (dustinle48) [inline README.md]: git pull origin **master**

File: README.md [hunk 3]
```diff
@@ -28,3 +28,16 @@ Open http://localhost:8080 -> use **root/root** to login -> import sql file in d
...
+
+8. Create new branch
+   git branch new_branch
```
Comment (dustinle48) [inline README.md]: git **checkout -b** new_branch

**Review by dustinle48 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dustinle48) [reply README.md]: And if the codes conflict, you need to ` git merge master` and resolve the conflict. Will show you later.

**Review by dustinle48 — APPROVED** [review]

File: online-grocery-store/src/components/layout/MainLayout.js (hunk 1 above)
Comment (LuongHuy) [reply online-grocery-store/src/components/layout/MainLayout.js]: fixed

File: README.md (hunk 2 above)
Comment (LuongHuy) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (LuongHuy) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -28,3 +28,16 @@ Open http://localhost:8080 -> use **root/root** to login -> import sql file in d
...
+   git push origin "branch_name"
+
+6. Change branch
+   git checkout "branch_name"
+
+7. Pull from main
+   git pull origin master
+
+8. Create new branch
+   git branch -b new_branch
```
Comment (dustinle48) [inline README.md:43]: git **checkout** -b new_branch

File: online-grocery-store/src/components/layout/MainLayout.js [hunk 5]
```diff
@@ -8,7 +9,20 @@ const Header = () => {
       <Grid.Col span={2} />
       <Grid.Col span={8}>
         <Flex h="100%" justify="space-between" align="center">
-          <Link href="/">Header</Link>
+          <Link href="/">TEACHER'S SHOP</Link>
+          <Link href="/">
+            <Image radius="md" src="Logo.jpg" h={50} w="auto" />
```
Comment (dustinle48) [inline online-grocery-store/src/components/layout/MainLayout.js:14]: Place Teacher's Shop here. Should wrap The logo and the name inside 1 <Link> only

**Pull request merged**